CairoMakie/src/CairoMakie.jl

```diff
@@ -1,3 +1,7 @@
+if get(ENV, "MAKIE_PRECOMPILE", "true") == "false"
+    __precompile__(false)
+end
```

Comment:
Doesn't have this to be set inside the module definition ?

Xref: https://github.com/timholy/SnoopCompile.jl/issues/324.

Reply:
It's documented that it disables precompilation for the whole file. I think I was told on Slack to do it like this but not completely sure

Reply:
I see.
Nitpicking, but these can be condensed to `__precompile__(get(ENV, "MAKIE_PRECOMPILE", "true") == "true")`.

Reply:
Thought about that but found it clearer to show that `__precompile__` is not called at all if the variable isn't set.

Reply:
Negating the logic would also allow to use any string (e.g. an Int which is shorter to type) to disable precompilation.

Reply:
maybe use `&&` then?